apps/electron-app/src/main/index.ts

```
@@ -5,7 +5,8 @@
 import { app, BrowserWindow, dialog, shell } from "electron";
 import { optimizer } from "@electron-toolkit/utils";
 import { config } from "dotenv";
-
+import { resolve } from "path";
```

ESLint

[error] 'resolve' is defined but never used.

(@typescript-eslint/no-unused-vars)